python/python/lance/dataset.py

```diff
@@ -4774,6 +4775,15 @@ def write_dataset(
         and can be retrieved using read_transaction().
         If both `commit_message` and `properties` are provided, `commit_message` will
         override any "lance.commit.message" key in `properties`.
+    data_bucket_uris: list of str, optional
+        *Experimental*. Additional bucket URIs for data file distribution.
```

Comment:
I think we don't need to say experimental here, but we might want to rename this feature flag to something more generic like `FLAG_BASE_PATHS`: https://github.com/lancedb/lance/pull/4552